### ISSUE TITLE
I/O logika by neměla být součástí doménových objektů (aplikační logiky), ne do constructorů

### ISSUE THREAD

**Issue by hakenr** [Slavek592/Bank#5]
https://github.com/Slavek592/Bank/blob/6d1c63ad06f9a9c56b07882ddd7b945188bd9421/Models/Account.cs#L23-L42

Načítání/ukládání/parsování/... se obvykle umisťuje mimo doménové objekty, ty by měly zapouzdřovat jenom aplikační logiku, aby nebyly závislé na konkrétním způsobu ukládání (výjimkou je návrhový vzor _Active Record_).

Zde bych doporučoval udělat samostatně třídu AccountPersister, nebo AccountLoader a zvenku to používat nějak jako
```csharp
var accountLoader = new AccountLoader();
var account = accountLoader.LoadAccount(filePath);
```

**Comment by Slavek592** [comment]
Ano, děkuji, teď, po takové době, nabírá tato rada další rozměry, když například input z konzole blokuje vytváření testů. :)
